src/amp/components/Pagination.tsx

```diff
@@ -50,7 +50,7 @@ export const Pagination: React.SFC<{
         url: string,
         icon: JSX.Element,
         suffix?: string,
-        hasRightMargin: boolean = false,
```

Comment:
I guess these types were removed because they're inferred to the default. Is this what we expect @oliverlloyd ?

Reply:
Ha, ha, ha. I love the mildly insulting lint message you get if you try to type `hasRightMargin`

![Screenshot 2020-01-14 at 13 54 23](https://user-images.githubusercontent.com/1336821/72349980-7493d200-36d5-11ea-81de-0d9af712cc1e.jpg)

Not entirely sure, I'm on board with `no-inferrable-types`

Reply:
Yeah, I'm happy to ignore that one. Will change

Reply:
done

Reply:
Cool. It's never good when you're left feeling like your linter has an attitude

Plus being explicit is nice